clcache.py

```diff
@@ -73,6 +83,12 @@
 Manifest = namedtuple('Manifest', ['includeFiles', 'hashes'])
 
 
+def printBinary(outstream, rawData):
+    with os.fdopen(outstream.fileno(), 'wb') as fp:
+        fp.write(rawData)
+        fp.flush()
```

Comment:
According to [this StackOverflow answer](http://stackoverflow.com/a/2447160/91757), closing a file implies flushing it - hence, I think the `fp.flush()` can be omitted.